cmd/loki/loki-docker-config.yaml

```diff
@@ -11,7 +11,6 @@ common:
       rules_directory: /loki/rules
   replication_factor: 1
   ring:
-    instance_addr: 127.0.0.1
```

Comment:
Q: Curious, what happens if we don't provide this config? Do Loki still choose `127.0.0.1` with random port?

Reply:
Loki will pick an address from the interface list, which from my experience in Docker is usually not 127.0.0.1, but some real IP on the docker network being used. Maybe this is on the case for compose or swarm?

Reply:
I meant in docker. Make sense :+1: